recipes/jsmin/meta.yaml

```diff
@@ -0,0 +1,35 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "2.2.1" %}
+
+package:
+    name: jsmin
+    version: {{ version }}
+
+source:
+    fn: jsmin-{{ version }}.tar.gz
+    url: https://files.pythonhosted.org/packages/87/8c/89cfe7ea967e0a4623b4e61008523ff40805c2bd4eabb1b07671643ea953/jsmin-2.2.1.tar.gz
```

Comment:
Please change it to `https://pypi.io/packages/source/j/jsmin/jsmin-{{ version }}.tar.gz and remove the comment in the first line and this is good to go!